### DISCUSSION TITLE
Large file not being ignored but in .gitattributes

### DISCUSSION THREAD

**Discussion by pmatos** [github-linguist/linguist#6329] [Classification] [answered]
Hi,

In my repo : https://github.com/pmatos/nora/blob/main/.gitattributes
I list expander/expander.rktl as linguistic-generated so that it's ignored. However, the repo is still marked as consisting mostly of code from this file.

I looked hard to see if I made a syntax mistake but can't see anything. Does anybody know what's going on?

**Comment by lildude** [comment, marked as answer]
You've got a typo:

```gitattributes
expander/expander.rktl linguistic-generated
                             # ^----- HERE. Drop the ic
```

**Reply by pmatos** [reply]
omg - I can't believe this. You're right. Thanks.
